src/library/StatBoxList/Pie.tsx

```diff
@@ -30,7 +29,20 @@ export const Pie = ({ label, stat, graph, tooltip, helpKey }: PieProps) => {
     <StatBox>
       <div className="content chart">
         <div className="chart">
-          <StatPie value={graph?.value1} value2={graph?.value2} />
+          <Chart
+            items={[
+              {
+                value: graph?.value1,
+                color: 'var(--accent-color-primary)',
+              },
+              {
+                value: graph?.value2,
+                color: 'var(--background-default)',
+              },
+            ]}
+            diameter={34}
+            speed={0.7}
```

Comment:
Actually no. Cause `duration` means that, the higher it is the loader the animation lasts;
In our case (`speed`) it is how fast it loads - meaning that the higher the `speed` the faster it animates.
e.g `0.5` speed is slower than `0.7`;

Reply:
Yes, I mean re-work the logic of the component so it accepts a duration prop instead of speed. Not critical but just more inline with how we do CSS transitions